10.5/debian-10/rootfs/opt/bitnami/scripts/libmariadb.sh

```diff
@@ -711,6 +711,9 @@ mysql_install_db() {
         command="${DB_BIN_DIR}/mysqld"
         args+=("--initialize-insecure")
     fi
+    if [[ -v SKIP_TEST_DB ]] && [[ "$SKIP_TEST_DB" == "true" ]]; then
+        args+=("--skip-test-db")
+    fi
```

Comment:
Some conventions for the env. variables used in Bitnami scripts:
- Prefix the env. variable with `MARIADB_`: `MARIADB_SKIP_TEST_DB`
- Please declare the env. variable in the [mysq-env.sh](https://github.com/bitnami/bitnami-docker-mariadb/blob/master/10.5/debian-10/rootfs/opt/bitnami/scripts/mariadb-env.sh)
- Use `yes/no` instead of `true/false` as allowed values.
```suggestion
    if is_boolean_yes "$MARIADB_SKIP_TEST_DB"; then
        args+=("--skip-test-db")
    fi
```
Other suggestions:
- Include this block under the `if [[ "$DB_FLAVOR" = "mariadb" ]]; then` block (line 708)
- You can use the block below to ensure that it's only executed if MariaDB is `10.5` or higher:
```bash
        if "$(mysql_get_version)" =~ ^10.[01234]. ]]; then
        (...)
        fi
```

Reply:
Hi, I made fix on my environment,

Does all this changes have to be made in all files (mysql-env.sh and libmariadb.sh) of all folder version of mariadb (folder 10.2/10.3...) because this feature should only be available in mariadb 10.5+ ?

Reply:
Yes @sebastienmusso
Please replicate the changes on every branch. You're right and it doesn't make sense to add a feature on branch `10.1` that's only available on `10.5` but our internal pipeline has a unique source of truth for this `libmariadb.sh` and, therefore, we need to replicate it.